lambda-alb/src/AWS/Lambda/Header.hs

```diff
@@ -1,33 +1,37 @@
 module AWS.Lambda.Header where
 
 import AWS.Lambda.Runtime.Prelude
+import Data.ByteString (ByteString)
 import Data.HashMap.Strict (HashMap)
 
 import qualified Data.Aeson           as JSON
 import qualified Data.Aeson.Types     as JSON
 import qualified Data.CaseInsensitive as CI
 import qualified Data.HashMap.Strict  as HashMap
 import qualified Data.Text.Encoding   as Text
-import qualified Network.HTTP.Types   as HTTP
 
-newtype Headers = Headers [HTTP.Header]
+type HeaderName = CI.CI Text
+
+type Header = (HeaderName, ByteString)
```

Comment:
I think we should not stop here and alos make the value a `Text` as ALB does not return byte strings.